app/controllers/webhooks/base_controller.rb

```diff
@@ -0,0 +1,49 @@
+# This is the base controller of Exercism's webhooks.
+#
+# It is intended to only be called from GitHub webhooks
+# and should not be called by any other source
+
+module Webhooks
+  class BaseController < ApplicationController
+    skip_before_action :verify_authenticity_token
+    skip_before_action :authenticate_user!
```

Comment:
Is this okay? We don't have need of a user or or CSRF tokens, as we do the signature validation. This is also how the SPI controller does it.

Reply:
Yep. This seems correct! :)